lib/streamlit/runtime/uploaded_file_manager.py

```diff
@@ -39,6 +39,16 @@ class UploadFileUrlInfo(NamedTuple):
     delete_url: str
 
 
+class DeletedFile:
+    """Return this from st.file_uploader and st.camera_input deserialize (so they can
+    be used in session_state), when widget value contains file record that is missing
+    from the storage.
+    DeleteFile instances filtered out before return final value to the user in script,
+    or before sending to frontend."""
+
+    pass
```

Comment:
Hm, not sure if this is necessary, but should we include the `fileId` in a `DeletedFile` so that we can differentiate between different files that are deleted?